### PR TITLE
login,signup pages

### PR DESCRIPTION
**Pull request by Hemasree5c2**

### CODE REVIEW

File: BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/controller/AuthController.java [hunk 1]
```diff
@@ -1,7 +1,63 @@
...
+            }
+        }
+        model.addAttribute("InvalidCredentials",true);
+        return "login";
+    }
+    @PostMapping("/signup")
+    public String append(@ModelAttribute(name="user") UserDTO user,Model model) {
+        String regexEmail="^(.+)@(.+)$";
+        Pattern pattern = Pattern.compile(regexEmail);
+        if(!pattern.matcher(user.getEmail()).matches()) {
```
Comment (ghost) [inline BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/controller/AuthController.java]: Check how to validate the request object correctly in spring

**Review by ghost — CHANGES_REQUESTED** [review]

File: BorrowMyProducts/pom.xml [hunk 2]
```diff
@@ -23,10 +23,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-security</artifactId>
-		</dependency>
+<!--		<dependency>-->
```
Comment (srivathsa-002) [inline BorrowMyProducts/pom.xml]: You can use spring security dependency by excluding SecurityAutoConfiguration.class in the main application

Comment (Hemasree5c2) [reply BorrowMyProducts/pom.xml]: okk..i will change it

**Review by vangaru-shanmukha — APPROVED** [review]

**Review by bshashank81 — COMMENTED** [review]
The code is fine, but all the logic of css and js are included along with the html. If the js, css and html are separated in different files, it increases the user readability.

**Review by NeerajaGoli — APPROVED** [review]

**Pull request merged**